distributed/client.py

```diff
@@ -1331,8 +1365,12 @@ def wait_for_workers(self, n_workers=0, timeout=None):
         timeout : number, optional
             Time in seconds after which to raise a
             ``dask.distributed.TimeoutError``
+        mode : WorkerWaitMode | str, optional
+            Mode to use when waiting for workers.
+            Default ``WorkerWaitMode.min`` or ``"at least"``, waits for at least ``n_workers``.
+            Other options include ``"at most"`` and ``"exactly"``.
```

Comment:
```suggestion
        mode : "at least" | "at most" | "exactly", optional
            Mode to use when waiting for workers.
            Default ``"at least"``, waits for at least ``n_workers``.
            One can also specify waiting for ``"at most"`` or ``"exactly"`` ``n_workers``.
```

(since workerwaitmode has gone), not sure the right way to type hint the literals in this case.